tests/Fixer/Import/OrderedImportsFixerTest.php

```diff
@@ -1279,6 +1279,65 @@ public function doSomething($foo)
         $this->doTest($expected, $input);
     }
 
+    public function testByLengthFixWithSameLengthAndCaseSensitive(): void
+    {
+        $this->fixer->configure([
+            'sort_algorithm' => OrderedImportsFixer::SORT_LENGTH,
+            'imports_order' => null,
+            'case_sensitive' => true,
+        ]);
+
+        $expected = <<<'EOF'
+            <?php
+
+            use Acme;
+            use BaRr;
+            use Bar1;
+            use Fooo;
+
+            class AnnotatedClass
+            {
+                /**
+                 * @Template(foobar=21)
+                 * @param Entity $foo
+                 */
+                public function doSomething($foo)
+                {
+                    $bar = $foo->toArray();
+                    /** @var ArrayInterface $bar */
+
+                    return function () use ($bar, $foo) {};
+                }
+            }
+            EOF;
+
+        $input = <<<'EOF'
+            <?php
+
+            use Acme;
+            use Fooo;
+            use Bar1;
+            use BaRr;
+
+            class AnnotatedClass
+            {
+                /**
+                 * @Template(foobar=21)
+                 * @param Entity $foo
+                 */
+                public function doSomething($foo)
+                {
+                    $bar = $foo->toArray();
+                    /** @var ArrayInterface $bar */
+
+                    return function () use ($bar, $foo) {};
+                }
+            }
```

Comment:
This looks like unneeded noise, fixer does not change it in any way so there's no point to keep it here. This code does not even use imported classes. Please simplify this test case to only necessary things (imports + empty class?).

Reply:
Maybe then clean up the rest of the test cases in this file?

Reply:
Got it, I will change only my code.

Reply:
Done